### PR TITLE
home task 03-05

### PR DESCRIPTION
**Pull request by MariaYakubovich**
Could You please check my home tasks.
Thank You in advance

### CODE REVIEW

File: mYakubovich/HTML_01/Python_01.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в
+		состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка
+		для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —
+		программирование графических интерфейсов (<u>GUI</u>). Давайте создадим простую программу прямо
+		сейчас.</p>
+		<H2>Наша первая программа</H2>
+		<p>Наберём код в текстовом редакторе (например, в <strong>gedit</strong>). Потом мы разберёмся, что делает каждая
+		строка и пойдём дальше.</p>
+		<p>Введите эти 4 строки:</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_01/Python_01.html]: Do not use  p tag for not human text

File: mYakubovich/HTML_01/Python_01.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+			#!/usr/bin/env python<br>
+			print 'Hello. I am a python program.'<br>
+			name = raw_input("What is your name?")<br>
+			print "Hello there, " + name + "!"</code></p>
+		<p>Это всё. Сохраните файл как <strong>hello.py</strong> в любом месте. Я предлагаю сохранить его в папке
+		<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+		Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+<pre><b><u><em>chmod +x hello.py</em></u></b></pre>
+		<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_01/Python_01.html]: Do not use  p tag for not human text

File: mYakubovich/HTML_01/Python_02.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+		этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше
+		программ.</p>
+		<H2>Списки</H2>
+		<p>Давайте рассмотрим другой тип переменных, который называется <i>списки</i>. В других 
+		языках программирования они называются <i>массивами</i>. Если мы обратимся к аналогии с
+		коробками для обуви, то массив (список) представляет собой несколько склеенных друг с
+		другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в 
+		другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы
+		запрограммируем его так:</p>
+<pre><br>
```
Comment (pluhin) [inline mYakubovich/HTML_01/Python_02.html]: It is not necessary to use here br, just add new line

File: mYakubovich/HTML_01/hometask_2.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+		Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в
+		состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка
+		для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —
+		программирование графических интерфейсов (GUI). Давайте создадим простую программу прямо
+		сейчас.</p>
+		<H2>Наша первая программа</H2>
+		<p>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая
+		строка и пойдём дальше.</p>
+		<p>Введите эти 4 строки:</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_01/hometask_2.html]: Do not use  p tag for not human text

File: mYakubovich/HTML_01/hometask_2.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+			#!/usr/bin/env python<br>
+			print 'Hello. I am a python program.'<br>
+			name = raw_input("What is your name?")<br>
+			print "Hello there, " + name + "!"</code></p>
+		<p>Это всё. Сохраните файл как hello.py в любом месте. Я предлагаю сохранить его в папке
+		python_examples в домашнем каталоге. Этот пример показывает, как просто написать программу на
+		Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+		<pre>chmod +x hello.py</pre>
+		<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_01/hometask_2.html]: Do not use  p tag for not human text

File: mYakubovich/HTML_02/Python_01_02.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+		состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка
+		для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —
+		программирование графических интерфейсов (<u>GUI</u>). Давайте создадим простую программу прямо
+		сейчас.</p>
+		<H2 id="programm">Наша первая программа</H2>
+		<p><A HREF="#contents">К содержанию</A></p>
+		<p>Наберём код в текстовом редакторе (например, в <strong>gedit</strong>). Потом мы разберёмся, что делает каждая
+		строка и пойдём дальше.</p>
+		<p>Введите эти 4 строки:</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_02/Python_01_02.html]: Do not use  p tag for not human text

File: mYakubovich/HTML_02/Python_01_02.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+			#!/usr/bin/env python<br>
+			print 'Hello. I am a python program.'<br>
+			name = raw_input("What is your name?")<br>
+			print "Hello there, " + name + "!"</code></p>
+		<p>Это всё. Сохраните файл как <strong>hello.py</strong> в любом месте. Я предлагаю сохранить его в папке
+		<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+		Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+<pre><b><u><em>chmod +x hello.py</em></u></b></pre>
+		<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_02/Python_01_02.html]: Do not use  p tag for not human text

File: mYakubovich/css_2/list.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+		<META charset="utf-8">
+		<TITLE>Лист</TITLE>
+		<style type="text/css">
+			ul{list-style: roman;
+			width; 350px;}
+			li ul{list-style: lower-roman;}
+			.code{background-color: black;
+			color: green;
+			width: 400px;
+			max-height: 4rem;}
```
Comment (pluhin) [inline mYakubovich/css_2/list.html]: also need to add ```line-height: 1rem;``` here

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by MariaYakubovich** [comment]
I'm very sorry but I have no idea how to resolve conflicts in fixed files.
Could You please help me.
Thank You in advance

**Comment by pluhin** [comment]
> I'm very sorry but I have no idea how to resolve conflicts in fixed files.
> Could You please help me.
> Thank You in advance

I cannot see conflicts in your code and github too, 
This "All checks have failed" - it is from external instrument, do not worry

**Pull request merged**